dashboard/src/actions/installedpackages.ts

```diff
@@ -234,11 +234,20 @@ export function installPackage(
     dispatch(requestInstallPackage());
     try {
       if (values && schema) {
-        const validation = validate(values, schema);
+        const schemaValidation = validateSchema(schema);
+        if (!schemaValidation.valid) {
+          const errorText = schemaValidation?.errors
+            ?.map(e => `  - ${e.instancePath}: ${e.message}`)
+            .join("\n");
+          throw new UnprocessableEntityError(
+            `The schema for this package is not valid. Please contact the package author. The following errors were found:\n${errorText}`,
+          );
+        }
```

Comment:
Validating the schema before actually creating the InstalledPackage. I have considered adding a "force button", however, Helm is also failing if a wrong schema is added (as it is validating against the `chart.values.schema.json` if present in the tar.gz). A "force" button would imply some extra backend work.

Currently, for us, the problem is mainly just https://github.com/bitnami-labs/readme-generator-for-helm/issues/34.